common/on-chip-calib.cpp

```diff
@@ -807,6 +824,12 @@ namespace rs2
                             "[0, 0.15) - Good\n"
                             "[0.15, 0.25) - Can be Improved\n"
                             "[0.25, ) - Requires Calibration");
+						    /* TODO : CHECK IF NEEDED
+						ImGui::SetTooltip("%s", "Calibration Health-Check captures how far camera calibration is from the optimal one\n"
```

Comment:
I think we can leave ranges positive, but please update values to reflect the correct ones (0.25, 0.75)